filament/backend/src/metal/MetalEnums.h

```diff
@@ -186,6 +188,25 @@ inline MTLPixelFormat getMetalFormat(TextureFormat format) noexcept {
     }
 #endif
 
+#if defined(IOS)
+    if (@available(iOS 8, macCatalyst 14, *)) {
+        switch (format) {
+            // EAC / ETC2 formats are only available on iPhone.
```

Comment:
Outdated comment, should be saying Apple GPUs.

Reply:
5c2c3efc2f8e3509c0c168d3e56a1c3f27386fb9